CesiumGeospatial/src/GlobeRectangle.cpp

```diff
@@ -142,6 +143,22 @@ GlobeRectangle::computeUnion(const GlobeRectangle& other) const noexcept {
       glm::max(this->_north, other._north));
 }
 
+glm::dvec2 GlobeRectangle::computeNormalizedCoordinates(
+    const Cartographic& cartographic) const noexcept {
+  double east = this->_east;
+  double cartoLong = cartographic.longitude;
+  if (east < this->_west) {
+    east += CesiumUtility::Math::TwoPi;
+    if (cartographic.longitude < 0) {
```

Comment:
This logic isn't quite right. If the west=179, east=10, and longitude=5 (all degrees), it's necessary to add 180 degrees to longitude in order to get the right answer, even though it's greater than 0.

I think you want to add 180 degrees to longitude whenever it is less than west.